### PR TITLE
Statuses

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: app/models/grda_warehouse/health_emergency/isolation.rb [hunk 1]
```diff
@@ -6,6 +6,13 @@
 
 module GrdaWarehouse::HealthEmergency
   class Isolation < GrdaWarehouseBase
-  include HealthEmergency
+    include HealthEmergency
```
Comment (awisema) [inline app/models/grda_warehouse/health_emergency/isolation.rb]: I don't think these are needed -- the prefix comes in from the module decl...

File: app/models/grda_warehouse/health_emergency/isolation.rb [hunk 2]
```diff
@@ -6,6 +6,13 @@
...
 
+    def status
+      return 'In Isolation' if start_date && end_date.blank?
+      return "In Isolation until #{end_date}" if start_date
```
Comment (awisema) [inline app/models/grda_warehouse/health_emergency/isolation.rb]: I remember discussing that we should list ended isolations... I don't remember where we came down... Should there be `"Was in Isolation until #{end_date}?"`

Comment (eanders) [reply app/models/grda_warehouse/health_emergency/isolation.rb]: Not sure yet, I think we can leave it as is, and if the date is in the past, then it should be clear they don't need to be there anymore.  We should ask these details on the call with Boston tomorrow.

File: app/models/grda_warehouse/health_emergency/isolation.rb (hunk 1 above)
Comment (eanders) [reply app/models/grda_warehouse/health_emergency/isolation.rb]: These bring in the acts_as_paranoid and relations, no the table prefix.

**Pull request merged**